examples/tf/rl2_ppo_halfcheetah_meta_test.py

```diff
@@ -0,0 +1,84 @@
+"""Example script to run RL2PPO meta test in HalfCheetah."""
+from garage import wrap_experiment
+from garage.envs.half_cheetah_vel_env import HalfCheetahVelEnv
```

Comment:
please use canonical imports here and below, e.g.`from garage.envs import HalfCheetahVelEnv` if it exists